### ISSUE TITLE
Question about duration loss

### ISSUE THREAD

**Issue by Wangzhen-kris** [jaywalnut310/glow-tts#55]
Hi, I trained the model with my own dataset. However, the duration loss increases gradually after a period of time for val data, it's overfitting, right? Can I stop the duration prediction after several epochs?
![image](https://user-images.githubusercontent.com/37279265/119604766-33134b00-be22-11eb-957e-f4607a357c3a.png)

**Comment by Selimonder** [comment]
Yeah, also in my experience duration predictor usually overfits quite fast. However beware of your validation set, if you have any abruptly stopped sentences.

Adding onto this question; Did anyone experimented with replacing duration predictor with FastSpeech2's?
